MiniTwit.Core/IRepositories/IMessageRepository.cs

```diff
@@ -9,4 +9,5 @@ public interface IMessageRepository
     Response<IEnumerable<Message>> GetAllNonFlagged();
     Response<IEnumerable<Message>> GetAllFollowedByUser(string userId);
     Response<Message> Create(string userId, string text);
+    User? GetUserByUsername(string username);
```

Comment:
Why is a user entry exposed in the repository?
MessageRepository should only return Messages.